android/titanium/libv8-services.js

```diff
@@ -42,6 +45,82 @@ async function loadPackageJson() {
 	return fs.readJson('../package.json');
 }
 
+/**
+ * Debug snapshot generation locally.
+ * @param {String} v8SnapshotHeaderFilePath The path to save generated snapshot header.
+ * @param {String} rollupFileContent Javascript content to store in snapshot.
+ */
+async function debugGenerateSnapshot(v8SnapshotHeaderFilePath, rollupFileContent) {
```

Comment:
<!--
  0 failure: 
  1 warning:  android/titanium/...
  
  
  DangerID: danger-id-Danger;
  File: android/titanium/libv8-services.js;
  Line: 53;
-->

- :warning: android/titanium/libv8-services.js line 53 – 'debugGenerateSnapshot' is defined but never used. (no-unused-vars)